rust/src/lib.rs

```diff
@@ -113,7 +113,6 @@ pub mod smb;
 pub mod krb;
 pub mod dcerpc;
 pub mod modbus;
-pub mod detect_parser;
```

Comment:
Should there not be `pub mod detect;` here ?

Reply:
There is on line 104.

Reply:
ok thanks

Reply:
That takes either a file or directory, ok :-)